src/bitcoin-tx.cpp

```diff
@@ -766,8 +767,6 @@ static std::string readStdin()
     if (ferror(stdin))
         throw std::runtime_error("error reading stdin");
 
-    boost::algorithm::trim_right(ret);
```

Comment:
why does this not have a replacement?

Reply:
It has: see L796 where it makes more sense :)